lib/browser/api/global-shortcut.js

```diff
@@ -1,5 +1 @@
-var globalShortcut
-
-globalShortcut = process.atomBinding('global_shortcut').globalShortcut
-
-module.exports = globalShortcut
+module.exports = process.atomBinding('global_shortcut')
```

Comment:
I think it should be `module.exports = process.atomBinding('global_shortcut').globalShortcut`?